CHANGELOG.rst

```diff
@@ -10,6 +10,8 @@ New Features
 
 - Use math operations like ``+-*/`` with py:class:`~climpred.classes.HindcastEnsemble`
   and :py:class:`~climpred.classes.PerfectModelEnsemble`. (:pr:`377`) `Aaron Spring`_.
+- Bias reduction py:class:`~climpred.classes.HindcastEnsemble.reduce_bias()`
```

Comment:
Maybe add more to this. Keep in mind this is for the users to read, not us. I usually edit them a lot before the version release but it would be helpful to handle that at this level, rather than at the end. This will be a nice big feature so we should advertise it as such 😄